modules/SofaMiscForceField/SofaMiscForceField_test/MeshMatrixMass_test.cpp

```diff
@@ -1669,17 +1668,9 @@ class MeshMatrixMass_test : public BaseTest
         EdgeSetTopologyModifier* modifier = root->getTreeObject<EdgeSetTopologyModifier>();
         ASSERT_NE(modifier, nullptr);
 
```

Comment:
In fact all the values below are the expected one. 
I would like to keep them for when the todo is done. Maybe move the //TODO line here at beginning and comment the whole block concerned.

Reply:
Done